ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+
+Thank you for being interested inv nvm, here is a quick form for problem reporting, please help us fill it if you are going to open an issue with bug/problem report, if you are just going to suggest feature or something else but not like this, please just remove it, thanks!
```

Comment:
I think this needs to be surrounded by HTML comments so that it shows up in the issue form, but won't show up in the posted issue. (also, `s/inv/in`, and `s/suggest feature/suggest a feature`, and `a/with bug/with a bug`)